ipatests/test_integration/test_caless.py

```diff
@@ -336,15 +338,15 @@ def create_pkcs12(cls, nickname, filename='server.p12', password=None):
 
         ipautil.run(["openssl", "pkcs12", "-export", "-out", filename,
                      "-inkey", key_fname, "-in", certchain_fname, "-passin",
-                     "pass:"+cls.cert_password, "-passout", "pass:"+password,
-                     "-name", nickname], cwd=cls.cert_dir)
+                     "pass:" + cls.cert_password, "-passout", "pass:" +
+                     password, "-name", nickname], cwd=cls.cert_dir)
 
     @classmethod
-    def prepare_cacert(cls, nickname):
+    def prepare_cacert(cls, nickname, filename):
```

Comment:
In almost all cases you are calling ``prepare_cacert`` with ``cls.pem_filename`` as argument. You can simplify the patch by keeping the current API: ``prepare_cacert`` with one argument defaults to ``cls.pem_filename``

```
@classmethod
def prepare_cacert(cls, nickname, filename=None):
    if filename is None:
        filename = self.pem_filename
```